cmd/satellite/main.go

```diff
@@ -177,58 +177,33 @@ func cmdDiag(cmd *cobra.Command, args []string) (err error) {
 	}()
 
 	//get all bandwidth agreements rows already ordered
-	baRows, err := database.BandwidthAgreement().GetAgreements(context.Background())
+	stats, err := database.BandwidthAgreement().GetTotals(context.Background(), time.Time{}, time.Now())
 	if err != nil {
 		fmt.Printf("error reading satellite database %v: %v\n", diagCfg.Database, err)
 		return err
 	}
 
 	// Agreement is a struct that contains a bandwidth agreement and the associated signature
-	type UplinkSummary struct {
-		TotalBytes        int64
-		PutActionCount    int64
-		GetActionCount    int64
-		TotalTransactions int64
-		// additional attributes add here ...
-	}
-
-	// attributes per uplinkid
-	summaries := make(map[storj.NodeID]*UplinkSummary)
-	uplinkIDs := storj.NodeIDList{}
-
-	for _, baRow := range baRows {
-		// deserializing rbad you get payerbwallocation, total & storage node id
-		rbad := baRow.Agreement
-		pbad := rbad.PayerAllocation
-		uplinkID := pbad.UplinkId
-		summary, ok := summaries[uplinkID]
-		if !ok {
-			summaries[uplinkID] = &UplinkSummary{}
-			uplinkIDs = append(uplinkIDs, uplinkID)
-			summary = summaries[uplinkID]
-		}
-
-		// fill the summary info
-		summary.TotalBytes += rbad.Total
-		summary.TotalTransactions++
-		switch pbad.GetAction() {
-		case pb.BandwidthAction_PUT:
-			summary.PutActionCount++
-		case pb.BandwidthAction_GET:
-			summary.GetActionCount++
-		}
-	}
+	const TotalBytes = 0
```

Comment:
Is there an single-source of truth enum that we could use instead of the magic numbers?